### PR TITLE
remove l5 repository. #141

### PR DESCRIPTION
**Pull request by chenghung**
#141

### CODE REVIEW

**Comment by chenghung** [comment]
[移除l5-Repository](https://trello.com/c/WMOVHM1M/607-%E7%A7%BB%E9%99%A4l5-repository)

File: Modules/Core/Repositories/BaseRepository.php [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+/**
+ * @template M of Model
+ * @implements Repository<M>
+ */
+abstract class BaseRepository implements Repository
+{
+    /**
+     * @return class-string
+     */
+    abstract public function model(): string;
```
Comment (sextitanic) [inline Modules/Core/Repositories/BaseRepository.php:22]: 還是這邊直接幫忙 App::make 然後 return 呢，節省其他 method 都做這個工

**Review by sextitanic — APPROVED** [review]
r+ with a comment, thanks.

File: Modules/Core/Repositories/BaseRepository.php [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+     */
+    public function all(): LazyCollection
+    {
+        $model = App::make($this->model());
+
+        return $model::lazy();
+    }
+
+    /**
+     * @param string|int $id
```
Comment (koshuang) [inline Modules/Core/Repositories/BaseRepository.php:35]: 建議跑過 composer refactor, composer phpcs:fix-all, insights:fix 看看有哪些 type hint comment 會被移除。

**Review by koshuang — APPROVED** [review]
r+ with a comment, thanks!

**Pull request merged**